.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,10 +6,11 @@
      commits if your PR already received reviews or comments.
 
      Before submitting a Pull Request, please ensure you've done the following:
-     - Read the DEV Contributing Guide and the Code of Conduct
-     - Provided tests for your changes
-     - Used descriptive commit messages
-     - Updated any relevant documentation and added any necessary screenshots
+     - 📖 Read the DEV Contributing Guide, https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md#create-a-pull-request and the Code of Conduct, https://github.com/thepracticaldev/dev.to/blob/master/CODE_OF_CONDUCT.md
```

Comment:
Can we not make DEV Contributing Guide a link in markdown, and the same with coc?

Reply:
I was going to do that, but this content is within an HTML comment and so it will only ever be seen in edit mode.

Reply:
Hah, I had the same comment, @Ridhwana !

Reply:
I think making a link is actually less readable, because this is inside of a comment. Right?

Reply:
I guess that's true, I couldn't tell that on initial review. How about this:
```
Before submitting a Pull Request, please ensure you've done the following:
- 📖 Read the DEV Contributing Guide: https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md#create-a-pull-request.
- 📖 Read the DEV Code of Conduct: https://github.com/thepracticaldev/dev.to/blob/master/CODE_OF_CONDUCT.md.
```

Reply:
Done